huaweicloud/services/acceptance/bms/resource_huaweicloud_bms_instance_test.go

```diff
@@ -209,6 +221,15 @@ resource "huaweicloud_bms_instance" "test" {
     tag2 = "value2"
   }
 
+  system_disk_type = "GPSSD"
+  system_disk_size = 150
```

Comment:
It is necessary to add `system_disk_type` and `system_disk_size` when testing `metadata`?